src/Components/ArticleByIdPage.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect } from "react";
+import { useParams, Link } from "react-router-dom";
+import axios from "axios";
+import ArticleDetails from "./ArticleByIdPageComponents/ArticleDetails";
+import Votes from "./ArticleByIdPageComponents/Votes";
+
+const ArticleByIdPage =()=>{
+
+    const [votes, setVotes] = useState(0)
+    const { article_id } = useParams();
+    const [isLoading, setIsLoading] = useState(true)
+    const [articleById, setArticleById] = useState({})
+
+    const articlesApi = axios.create({
+        baseURL: "https://v2-nc-news-backend-project-northcoders.onrender.com"
+    })
+
+    useEffect(()=>{
+        articlesApi.get(`/api/articles/${article_id}`).then(({data})=>{
+             setIsLoading(false)
```

Comment:
Great to see the use of an axios instance, but this should take place in a separate api.js file to separate your concerns nicely. Also, for each bit of api functionality like a getArticleByID, it's ideal to have a function in your api.js that handles it, which you can then require & invoke anywhere.

Reply:
done!